tools/docker/coverage.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+###############################################################
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# SPDX-FileCopyrightText: 2019-2020 the prplMesh contributors (see AUTHORS.md)
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+###############################################################
+
+scriptdir="$(cd "${0%/*}" || exit 1; pwd)"
+rootdir="${scriptdir%/*/*}"
+
+# shellcheck source=functions.sh
+. "${rootdir}/tools/functions.sh"
+
+main() {
+    image=${PRPLMESH_BUILDER_IMAGE-"${DOCKER_REGISTRY}"prplmesh-builder}
+    # Default docker arguments
+    docker_args=(
+        --workdir "${rootdir}"
+        --user "${SUDO_UID:-$(id -u)}:${SUDO_GID:-$(id -g)}"
+        -e "USER=${SUDO_USER:-${USER}}"
+        -v "${rootdir}:${rootdir}"
+        --name prplMesh-coverage
+        --entrypoint "./coverage.sh"
```

Comment:
gcovr is pretty stable, so I see little reason to add this docker wrapper script... Then again, it doesn't hurt much to add it.

Reply:
I'm not quite sure to understand your comment. 

You wouldn't believe the bureaucracy required to install gcovr in my development environment. However, I have no problem in my company with creating and running a Docker image with gcovr inside. 

We might reconsider using the lighter Alpine-based image instead of the builder image to create coverage reports faster. Or better off, have it in both so one can decide to run the builder image in interactive mode and then perform compile-test-coverage cycles even faster.

Reply:
> You wouldn't believe the bureaucracy required to install gcovr in my development environment.

Ah, yes, I always forget that.